src/rt/dmain2.d

```diff
@@ -197,7 +198,7 @@ extern (C) int rt_init()
         // in other druntime systems.
         _d_initMonoTime();
         thread_init();
-        gc_init();
+        // TODO: fixme - calls GC.addRange -> Initializes GC
```

Comment:
Yes, that's what needs to be fixed, e.g. by storing the ranges somewhere else or in ProtoGC until we lazily initialize the real GC.

Reply:
@MartinNowak what use cases is this serving? The win of ProtoGC is to get away without ever initializing the GC. But it's unlikely someone adds a bunch of ranges of interest to the GC but ends up doing no allocation. Also: Where would the array be allocated? Also: this has gotten really long in the teeth, improvements can be discussed later.